src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -206,7 +206,7 @@ static void CheckForExperimentStatus(ITextView textView, Document document)
         {
             // Have to store the trigger location to reuse it to get expanded items and also reuse the snapshot included in 
             // some projections related scenarios where data and session in further calls are able to provide other snapshots.
-            session.Properties.AddProperty(TriggerLocation, triggerLocation);
+            session.Properties[TriggerLocation] = triggerLocation;
```

Comment:
I will send you an update where this data is recorded in `IAsyncCompletionSessionOperations.InitialTriggerLocation`

Reply:
Fixed. Thanks!